extensions/wikia/AchievementsII/Ach_setup.php

```diff
@@ -1,199 +1,199 @@
 <?php
-define('ACHIEVEMENTS_ENABLED', true);
+define( 'ACHIEVEMENTS_ENABLED', true );
 
-define('ACHIEVEMENTS_BADGE_PREFIX', 'badge-');
-define('ACHIEVEMENTS_HOVER_PREFIX', 'achievements-badge-hover-');
+define( 'ACHIEVEMENTS_BADGE_PREFIX', 'badge-' );
+define( 'ACHIEVEMENTS_HOVER_PREFIX', 'achievements-badge-hover-' );
 
 // BADGE LEVELS
-define('BADGE_LEVEL_BRONZE', 1);
-define('BADGE_LEVEL_SILVER', 2);
-define('BADGE_LEVEL_GOLD', 3);
-define('BADGE_LEVEL_PLATINUM', 4);
+define( 'BADGE_LEVEL_BRONZE', 1 );
+define( 'BADGE_LEVEL_SILVER', 2 );
+define( 'BADGE_LEVEL_GOLD', 3 );
+define( 'BADGE_LEVEL_PLATINUM', 4 );
 
 // BADGE TYPE ID's
-define('BADGE_WELCOME', -1);
-define('BADGE_INTRODUCTION', -2);
-define('BADGE_SAYHI', -3);
-define('BADGE_CREATOR', -4);
-define('BADGE_POUNCE', -5);
-define('BADGE_CAFFEINATED', -6);
-define('BADGE_LUCKYEDIT', -7);
-define('BADGE_EDIT', -8);
-define('BADGE_PICTURE', -9);
-define('BADGE_CATEGORY', -10);
-define('BADGE_BLOGPOST', -11);
-define('BADGE_BLOGCOMMENT', -12);
-define('BADGE_LOVE', -13);
-define('BADGE_SHARING', -14);
-
-define('BADGE_TYPE_INTRACKEDITPLUSCATEGORY', 1); // EDIT+CATEGORY
-define('BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM', 2); // COMMUNITY CUSTOM
-define('BADGE_TYPE_NOTINTRACKSTATIC', 3); // STATIC IN TRACK
-define('BADGE_TYPE_INTRACKSTATIC', 4); // STATIC NOT IN TRACK
-
-$wgExtensionCredits['other'][] = array(
-	'path'           => __FILE__,
-	'name'           => 'AchievementsII',
-	'author'         => 'Wikia',
+define( 'BADGE_WELCOME', -1 );
+define( 'BADGE_INTRODUCTION', -2 );
+define( 'BADGE_SAYHI', -3 );
+define( 'BADGE_CREATOR', -4 );
+define( 'BADGE_POUNCE', -5 );
+define( 'BADGE_CAFFEINATED', -6 );
+define( 'BADGE_LUCKYEDIT', -7 );
+define( 'BADGE_EDIT', -8 );
+define( 'BADGE_PICTURE', -9 );
+define( 'BADGE_CATEGORY', -10 );
+define( 'BADGE_BLOGPOST', -11 );
+define( 'BADGE_BLOGCOMMENT', -12 );
+define( 'BADGE_LOVE', -13 );
+define( 'BADGE_SHARING', -14 );
+
+define( 'BADGE_TYPE_INTRACKEDITPLUSCATEGORY', 1 ); // EDIT+CATEGORY
+define( 'BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM', 2 ); // COMMUNITY CUSTOM
+define( 'BADGE_TYPE_NOTINTRACKSTATIC', 3 ); // STATIC IN TRACK
+define( 'BADGE_TYPE_INTRACKSTATIC', 4 ); // STATIC NOT IN TRACK
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)